geoopt/tensor.py

```diff
@@ -118,7 +129,7 @@ def __reduce_ex__(self, proto):
         return _rebuild_manifold_parameter, proto + (self.manifold,)
 
     @insert_docs(Manifold.unpack_tensor.__doc__, r"\s+tensor : .+\n.+", "")
-    def as_point(self):
+    def as_point(self) -> Union[torch.Tensor, Tuple[torch.Tensor]]:
```

Comment:
I don't see any usage of this method in code base.

Reply:
Thanks, this was fixed and I added a test.